checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInference.java

```diff
@@ -128,6 +141,13 @@ public class MustCallInference {
   /** The element referring to the enclosing class of the current method. */
   private final @Nullable TypeElement classElt;
 
+  /**
+   * This map is used to track alias relationships between return nodes and method parameters. The
+   * keys are the obligation of return nodes, and the values are the index of current method formal
+   * parameter (0-based) that is alias with the return node.
```

Comment:
WPI has switched to 1-based indexing for the parameters (with 0 representing the receiver), and I think we should do the same here for consistency.